android/AndroidManifest.xml

```diff
@@ -25,6 +25,7 @@
   <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION"/>
   <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
   <uses-permission android:name="android.permission.ACCESS_LOCATION_EXTRA_COMMANDS" />
+  <uses-permission android:name="android.permission.ACCESS_BACKGROUND_LOCATION" />
```

Comment:
Is this really necessary to ask for this permission?
This can really complicate things

Reply:
Yes, ACCESS_BACKGROUND_LOCATION permission is needed for the background location ;)